build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts

```diff
@@ -54,7 +54,7 @@ export function getCommentForCodeCoverageDiff(
 		getSummaryFooter(baselineBuildInfo),
 		success
 			? "### Code coverage comparison check passed!!"
-			: "### Code coverage comparison check failed!!",
+			: "### Code coverage comparison check failed!!<br>More Details: [Readme](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/docs/codeCoverageDetails.md#success-criteria)",
```

Comment:
Since the content that used to be here is now inlined in the caller of this function depending on the value of `success` (outside the scope of this function), seems like we don't need to have a condition here anymore, just return the heading in all cases, and we can remove the `success` parameter?

Reply:
I can remove the extra content, but there is no point of having all that info if there is no regression. It will unnecessarily post such a big comment on the PR.

Reply:
We still need the success parameter, because we use that below to decide what to return.

Reply:
I think I still want to keep this part here which is at least telling the whether the thing passed or not and the name of the file is also about getting the comment. 
The rest caller can add which we are doing, as per the behavior the caller wants in the PR.

Reply:
Yep, I was thinking the link pointed to something else.